libs/designsystem/src/lib/components/toggle-button/toggle-button.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
+import { first } from 'rxjs/operators';
+
+import { ToggleButtonComponent } from './toggle-button.component';
+
+describe('ToggleButtonComponent', () => {
+  let spectator: Spectator<ToggleButtonComponent>;
+  const createComponent = createComponentFactory(ToggleButtonComponent);
+
+  beforeEach(() => (spectator = createComponent()));
+
+  it('should should toggle checked state on click', (done) => {
```

Comment:
Consider testing for both the event emitter (checkChanged) and the state (checked)

Reply:
Both get covered here?

Reply:
Nope, you only expect the event being emitted, you don't actually test the state (`checked === false`)